internal/pkg/integration/testdata/variables/test.yaml

```diff
@@ -27,3 +27,17 @@ run:
   - name: validate
     runner: validate
     expect: success
+  - name: eval-local
+    runner: eval
+    platform: linux/amd64
```

Comment:
any reason explicit platform was specified, or does this not work with arm64?

Reply:
good question, I think it is even ignored actually or not used, I will double check and remove it